src/v2/views/RevokeView.tsx

```diff
@@ -1,18 +1,19 @@
-import React from "react";
+import React, { useState } from "react";
 import { observer } from "mobx-react";
 import Layout from "../components/core/Layout";
 import H1 from "../components/ui/H1";
 import { T } from "src/renderer/i18n";
 import { Select, SelectOption } from "../components/ui/Select";
 import { useStore } from "../utils/useStore";
+import { Address } from "src/interfaces/keystore";
 import Button from "../components/ui/Button";
-import { ipcRenderer } from "electron";
 import { useHistory } from "react-router";
 
 const transifexTags = "v2/revoke-view";
 
 function RevokeView() {
   const account = useStore("account");
+  const [address, setAddress] = useState<Address>(account.keyring[0].address);
```

Comment:
It assumes users will come to `RevokeView` when only they have an address at leat, right?